api/common.go

```diff
@@ -15,11 +15,11 @@ import (
 	"github.com/docker/libtrust"
 )
 
+// Version of Current REST API
+var Version = version.Version("1.22")
```

Comment:
Is relying on setting a package global a good method of doing this? Shouldn't this be daemon field?